docs/postman/sample_requests/sample_interaction.json

```diff
@@ -25,7 +25,7 @@
           {
             "idType": "URI",
             "type": "GlobalReference",
-            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/",
+            "value": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/sampleSM",
```

Comment:
remove /sampleSM

Reply:
reverted it to the original value without /sampleSM